relays/ethereum/src/ethereum_client.rs

```diff
@@ -122,11 +190,7 @@ pub async fn transactions_receipts(
 ) -> (Client, Result<(EthereumHeaderId, Vec<Receipt>), Error>) {
 	let mut transactions_receipts = Vec::with_capacity(transacactions.len());
 	for transacaction in transacactions {
```

Comment:
There are typos here: 
```suggestion
	for transaction in transactions {
```
but it's a minor thing :)